docs/VehicleInfo/GetVehicleType/index.md

```diff
@@ -47,13 +46,14 @@ GetVehicleType
   "jsonrpc" : "2.0",
   "result" :
   {
-    "vehicleType" :
-    [
-      "make" : "Ford",
-      "model" : "Fusion",
-      "modelYear" : "2013",
-      "trim" : "SE"
-    ]
+    "vehicleType" :[
```

Comment:
This isn't an array in the HMI_API

Reply:
Nice catch 👀 . Resolved in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/155/commits/f84f198b7223d6940c1d2103ca3965f8d29ed8a4